index.html

```diff
@@ -2766,134 +2729,118 @@ <h3>Consumption</h3>
 suggestions in section 3.9 of the CBOR [[RFC7049]] specification for
 deterministic mappings the following constraints of the DID Document model MUST
 be followed:
-        </p>
+      </p>
         <ul>
-          <li>
-The keys in every map must be sorted lowest value to highest. Sorting is
-performed on the bytes of the representation of the keys.
-          </li>
-          <li>
-Indefinite-length items must be made into definite-length items.
-          </li>
-        </ul>
+  <li>Duplicate key in the same CBOR map MUST throw an Error.</li>
+  <li>All member properties of a CBOR encoded DID document MUST be either represented in the `@context`, be present in the 
```

Comment:
That this CBOR representation actively supports an `@context` property is strange to me.